script/openqa-bootstrap

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -x
+
+zypper -n install --no-recommends openQA-local-db apache2 openQA-worker qemu-kvm
+
+# setup database
+systemctl enable --now postgresql
+su postgres -c "createuser -D geekotest"
+su postgres -c "createdb -O geekotest openqa"
+
+
+# setup webserver and fake-auth
+# from script/setup-single-instance (https://github.com/os-autoinst/openQA/pull/1933)
+for i in headers proxy proxy_http proxy_wstunnel rewrite ; do a2enmod $i ; done
+sed -i -e 's/^.*httpsonly.*$/httpsonly = 0/g' /etc/openqa/openqa.ini
+sed -i -e 's/#.*method.*OpenID.*$/&\nmethod = Fake/' /etc/openqa/openqa.ini
+sed "s/#ServerName.*$/ServerName $(hostname)/" /etc/apache2/vhosts.d/openqa.conf.template > /etc/apache2/vhosts.d/openqa.conf
+
+
+# fetch tests and needles
+echo "opensuse ..."
+if ping -c1 gitlab.suse.de. ; then
+	# use faster local mirror if run from within SUSE network
+	export needles_giturl="https://gitlab.suse.de/openqa/os-autoinst-needles-opensuse-mirror.git"
+fi
+/usr/share/openqa/script/fetchneedles
+
+if ping -c1 gitlab.suse.de. ; then
```

Comment:
In this case I would just call https://gitlab.suse.de/openqa/scripts/blob/master/fetchneedles-sles instead

Reply:
It did nothing for the sle needles.
Also I don't see why I should need an additional script for a simple `git clone …; chown …`.

Reply:
It's not just that as it also does the normal "fetchneedles" call